toolkit/scripts/imggen.mk

```diff
@@ -5,7 +5,8 @@ $(call create_folder,$(IMAGEGEN_DIR))
 
 # Resources
 config_name              = $(notdir $(CONFIG_FILE:%.json=%))
-config_other_files       = $(if $(CONFIG_FILE),$(call shell_real_build_only, find $(CONFIG_BASE_DIR)))
+CONFIG_BASE_DIR          = $(dir $(CONFIG_FILE:%.json=%))
```

Comment:
This could probably be moved into the script, do a `config_file_base_dir=$(dirname "$config_file")`